### PR TITLE
WL-470: Checkout buttons center aligned on all screens.

### PR DESCRIPTION
**Pull request by malikshahzad228**

### CODE REVIEW

**Comment by clintonb** [comment]
All UI changes should be accompanied by screenshots. In this case, we need to see shots for mobile, tablet, and desktop.

**Comment by malikshahzad228** [comment]
Screenshots:

screen shot 2016-05-17 at 3 35 51 pm

screen shot 2016-05-17 at 5 18 41 pm

screen shot 2016-05-17 at 5 19 58 pm

**Comment by malikshahzad228** [comment]
@clintonb @mattdrayer FYI

File: ecommerce/templates/oscar/basket/partials/basket_content.html [hunk 1]
```diff
@@ -116,35 +116,37 @@
...
-                                {# Translators: Do NOT translate the name PayPal. #}
-                                {% trans "Checkout with PayPal" %}
-                            {% endif %}
-                        </button>
-                    {% endfor %}
-                {% endif %}
+            <div class="col-lg-6 col-lg-offset-6 col-md-8 col-md-offset-4 col-sm-10 col-sm-offset-2">
+                <div class="pull-right payment-buttons" data-basket-id="{{ basket.id }}">
+                    {% if free_basket %}
+                    <a  href="{% url 'checkout:free-checkout' %}"
```
Comment (mattdrayer) [inline ecommerce/templates/oscar/basket/partials/basket_content.html:122]: Tiny Nit: Remove extra space in `<a  href`

**Comment by mattdrayer** [comment]
LGTM -- one tiny nit that does not affect the operation of the code -- 👍  after rebase + green build

**Pull request merged**